main.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable linebreak-style */
+
+class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+
+  static add(book) {
+    const books = Book.getAll();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static remove(id) {
+    const books = Book.getAll();
+    const index = books.findIndex((book) => book.id === id);
+    if (index >= 0) {
+      books.splice(index, 1);
+      localStorage.setItem('books', JSON.stringify(books));
+    }
+  }
+
+  static getAll() {
+    return JSON.parse(localStorage.getItem('books')) || [];
+  }
+
+  static displayAll() {
+    const list = document.querySelector('#list');
+    list.innerHTML = '';
+    Book.getAll().forEach((book) => {
+      const listItem = document.createElement('li');
+      listItem.innerHTML = `"${book.title}" by ${book.author}`;
+
+      // Create a Remove button for each list item
+      const removeButton = document.createElement('button');
+      removeButton.textContent = 'Remove';
+      removeButton.addEventListener('click', () => {
+        Book.remove(book.id);
+        listItem.remove();
+      });
+
+      listItem.appendChild(removeButton);
+      list.appendChild(listItem);
+    });
+  }
+}
+
+Book.displayAll();
```

Comment:
- I suggest calling this method using an event listener, this would make your code look more readable. Please fix this 🙏

Reply:
Above function is also there in the event listener  kindly refer to the screen shot below.
![image](https://user-images.githubusercontent.com/117091923/220427355-514a5dbe-9d40-4ace-8a1e-84766d8374a0.png)
This function call above is for page refresh now we have put this function call into window.onload event to display the data when page refreshes.
All suggested changes have been made and sending for a review again.